librz/core/cmd_api.c

```diff
@@ -255,14 +255,14 @@ RZ_API void rz_cmd_batch_end(RzCmd *cmd) {
 	sort_groups(rz_cmd_get_root(cmd));
 }
 
-static RzOutputMode suffix2mode(const char *suffix) {
+RZ_IPI RzOutputMode rz_char_to_output_mode(const char *suffix) {
 	size_t i;
 	for (i = 0; i < RZ_ARRAY_SIZE(argv_modes); i++) {
 		if (!strcmp(suffix, argv_modes[i].suffix)) {
 			return argv_modes[i].mode;
 		}
 	}
-	return 0;
+	return *suffix;
```

Comment:
why? I think we should return a default value, which can either be `RZ_OUTPUT_MODE_STANDARD` or a new value `RZ_OUTPUT_MODE_UNKNOWN`.

Reply:
I thought `return *suffix` would be similar to the default case in the original `rz_char_to_output_mode` function. This would take care of the case when the character is `.`, `,`, etc that isn't in `argv_modes[]`.

Reply:
I think it would make sense to have `RZ_OUTPUT_MODE_UNKNOWN` instead to take care of such characters.

Reply:
I agree with introducing the `RZ_OUTPUT_MODE_UKNOWN`. We also should check it never happens in the code with macroses like `rz_return_val_if_fail(mode != RZ_OUTPUT_MODE_UNKNOWN)` or `rz_warn_if_reached()`.

Reply:
> I thought `return *suffix` would be similar to the default case in the original `rz_char_to_output_mode` function. This would take care of the case when the character is `.`, `,`, etc that isn't in `argv_modes[]`.

I think those cases should be handled before calling this function or only if this function returns RZ_OUTPUT_MODE_UNKNOWN. I don't like doing `return *suffix` because `*suffix` may be any character and a character is not of type `RzOutputMode`. This function should return a `RzOutputMode`.